spec/rollbar/sidekiq_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+
+unless RUBY_VERSION == '1.8.7'
+  require 'sidekiq'
+  require 'rollbar/sidekiq'
+end
+
+describe "Sidekiq integration", :reconfigure_notifier => false do
+  before{ skip if RUBY_VERSION == '1.8.7' }
+
+  describe "Rollbar::Sidekiq.handle_exception" do
+    let(:msg_or_context) { ["hello", "error_backtrace", "backtrace", "goodbye"] }
+    let(:exception) { StandardError.new("oh noes") }
+    let(:rollbar) { double }
+    subject do
+      Rollbar::Sidekiq.handle_exception(msg_or_context, exception)
```

Comment:
I prefer to see the 'trigger' statement in the `it` block. And subject should be related to the object you are testing. In this case `described_class` will return `Rollbar::Sidekiq`.

You can change this to `subject { described_class }`. And use later `subject.handle_exception(msg_or_context, exception)` in the `it` block.